Documentation/observability/metrics.rst

```diff
@@ -1271,3 +1271,24 @@ which is passed as the ``controller-group-metrics`` configuration
 flag. The current default set for ``kvstoremesh`` found in the
 Cilium Helm chart is the special name "all", which enables the metric
 for all controller groups. The special name "none" is also supported.
+
+NAT
+~~~
+
+======================================== ================================================== ========== ========================================================
+Name                                     Labels                                             Default    Description
+======================================== ================================================== ========== ========================================================
+``nat_endpoint_max_connection``          ``family``                                         Enabled    Saturation of the most the most saturated distinct NAT mapped connection, in terms of egress-IP and remote endpoint address.
```

Comment:
Should be: 
```
Saturation of the most saturated distinct NAT mapped connection
```

Instead of:
```
Saturation of the most the most saturated distinct NAT mapped connection
```

Reply:
Oh nice catch 😄  thanks